docs/mp/faulttolerance/01_overview.adoc

```diff
@@ -22,6 +22,8 @@
 :description: Fault Tolerance Introduction
 :keywords: helidon, webserver, faulttolerance, mp
 :fault-tolerance-spec-url: https://github.com/eclipse/microprofile-fault-tolerance
+:scheduled-executor-config: https://helidon.io/docs/v2/apidocs/io.helidon.common.configurable/io/helidon/common/configurable/ScheduledThreadPoolSupplier.Builder.html#config(io.helidon.config.Config)
```

Comment:
There is a property for referencing javadocs: `{javadoc-base-url}`

Reply:
Right, I'll fix that.